modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,42 @@
+= Introduction to data migration
+:page-tag: migration,zdm,zero-downtime,zdm-proxy,introduction
+ifdef::env-github,env-browser,env-vscode[:imagesprefix: ../images/]
+ifndef::env-github,env-browser,env-vscode[:imagesprefix: ]
+
+Enterprises today want to reliably migrate mission-critical client applications and data to cloud environments with zero downtime or near zero downtime during the migration. 
+
+A migration is a workflow that encompasses the lifecycle of uploading and importing your data to the selected keyspaces. 
+DataStax has developed a set of thoroughly tested self-service tools to walk you through well-defined migration options:
+
+* Zero Downtime Migration
+* Cassandra Data Migrator
+* DSBulk Migrator
+ 
+With these tools, you can move your application to {astra_db}, DSE, or Cassandra with no downtime and with minimal configuration changes.
+Your clusters are kept in sync at all times by a dual-write logic configuration.
+You can roll back at any point, for complete peace of mind.
+
+== Migration process
+
+Downtime may be necessary when you need to migrate data that is critical for ongoing operations. 
+In some cases, the migration process itself may require systems to be taken offline temporarily. 
+Your business determines the downtime for any migration and how much downtime (if any) is necessary. 
+Large amounts of data require more time; small amounts of data require less time and may also be migrated through the {astra_ui}.
+
+== Data migration tools
+
+For modest data volumes where temporary downtime is tolerable, Datastax recommends two open-source data migration tools to help with brief outages:
+
+* *Cassandra Data Migrator*: The best choice to migrate large data quantities, and where detailed logging, data verifications, table column renaming (if needed), and reconciliation options are provided.
+* *DSBulk Migrator*: This leverages DataStax Bulk Loader (DSBulk) to perform the data migration, and provides new commands specific to migrations. DSBulk Migrator is ideal for simple migration of smaller data quantities, and where data validation (other than post-migration row counts) is not necessary.
+
+These tools provide sophisticated features that help you migrate your data from any Cassandra Origin (Apache Cassandra®, {company} Enterprise (DSE), {company} {astra_db}) to any Cassandra Target (Apache Cassandra, DSE, {company} {astra_db}).
+
+For large datasets, it's best to use the Zero Downtime Migration (ZDM).
```

Comment:
ZDM is available for small and large data sets.  ZDM has two components
1. keeping the two databases in sync with ZDM proxy.  The proxy does the dual writing to both clusters.
2. data migration.  This step is necessary for any migration and we have CDM and dsbulk and soon Astra DB Sideloader when it GAs.  The data migration is part of the zero-downtime migration.  I'm not sure why we're separating ZDM from CDM/DSbulk.

Reply:
I do not understand the rationale for which we seem to recommend ZDM only for large datasets. ZDM can be used for migrations of any size and should be our go-to approach, while downtime migrations should be considered the exception.